lua/spectre_common.lua

```diff
@@ -129,14 +129,20 @@ local function determine_if_cacheable(url, namespace, request_headers)
         refresh_cache = false,
     }
 
-    local spectre_config = config_loader.get_spectre_config_for_namespace(namespace)
-    if spectre_config == nil then
+    local spectre_config = config_loader.get_spectre_config_for_namespace('casper.internal')
```

Comment:
let's use the CASPER_INTERNAL_NAMESPACE since it's an internal hardcoded constant.